Languages/Danish/Keyed/Multiplayer.xml

```diff
@@ -0,0 +1,132 @@
+  <MpSend>Send</MpSend>
+  <MpSteamAcceptTitle>Accepter Steam brugere</MpSteamAcceptTitle>
+  <MpSteamAcceptDesc>Click for at acceptere</MpSteamAcceptDesc>
+  <MpSteamAccepted>Bruger er blevet accepteret</MpSteamAccepted>
+  
+  <MpConnectingTo>Tilslutter til {0}:{1}</MpConnectingTo>
+  <MpConnected>Tilsluttet.</MpConnected>
+  <MpDownloading>Downloader ({0}%)</MpDownloading>
+  
+  <MpSteamConnectingTo>Tilslutter til et spil startet af {0}</MpSteamConnectingTo>
+  <MpSteamConnectingWaiting>Venter på at værten acceptere</MpSteamConnectingWaiting>
+  
+  <MpTicksBehind>Du er {0} ticks bagud.</MpTicksBehind>
+  <MpLowerGameSpeed>Overvej at sænke spillets hastighed.</MpLowerGameSpeed>
+  <MpForceCatchUp>Klik for at tvinge indhentelse.</MpForceCatchUp>
+  
+  <MpDesynced>Spillet har desynkroniseret.</MpDesynced>
+  <MpTryResync>Prøv på at resynkronisere</MpTryResync>
+  
+  <MpSaveReplayAs>Gem replay som</MpSaveReplayAs>
+  <MpWillOverwrite>Vil overskrive</MpWillOverwrite>
+  <MpReplaySaved>Replay gemt</MpReplaySaved>
+  
+  <MpGameName>Spil navn</MpGameName>
+  <MpMaxPlayers>Maks antal spillere</MpMaxPlayers>
+  <MpAutosaveEvery>Auto-save hver</MpAutosaveEvery>
+  <MpAutosaveMinutes>dage</MpAutosaveMinutes>
+  <MpLanDesc1>Transmiter spillet på dit lokale netværk.</MpLanDesc1>
+  <MpLanDesc2>Løste LAN addresse: {0}</MpLanDesc2>
+  <MpArbiterDesc>En seperat process af spillet som køre i baggrunden og hjælper med at løse problemer relateret til desynkronisering.</MpArbiterDesc>
+  <MpAsyncTimeDesc>Separat tidskontrol for planeten og hver koloni.</MpAsyncTimeDesc>
+  <MpExperimentalFeature>Eksperimental funktion</MpExperimentalFeature>
+  
+  <MpHostButton>Host</MpHostButton>
+  <MpWatchReplay>Se på</MpWatchReplay>
+  <MpNothingSelected>Intet markeret</MpNothingSelected>
+  <MpNoSaves>Multiplayer kan kun afspilles fra allerede eksisterende spil/saves</MpNoSaves>
+  
+  <MpHostIngame>Host dette spil</MpHostIngame>
+  <MpHostReplay>Host et replay</MpHostReplay>
+  <MpHostSavefile>Host en gemt fil</MpHostSavefile>
+  
+  <MpJoinButton>Tilslut</MpJoinButton>
+  <MpConnectButton>Opret forbindelse</MpConnectButton>
+  <MpLanSearching>Søger</MpLanSearching>
+  
+  <MpNotConnectedToSteam>Ikke forbundet til Steam</MpNotConnectedToSteam>
+  <MpNoFriendsPlaying>Ingen venner spiller på nuværende tidspunkt RimWorld</MpNoFriendsPlaying>
+  <MpPlayingRimWorld>Spiller RimWorld</MpPlayingRimWorld>
+  <MpNotInMultiplayer>Ikke i multiplayer</MpNotInMultiplayer>
+  
+  <MpPauseOnAutosave>Pauser spillet når der auto-saves</MpPauseOnAutosave>
+  <MpAppendNameToAutosave>Tilføj et navn til auto-save</MpAppendNameToAutosave>
+  <MpPauseAutosaveCounter>Pauser auto-save tælleren</MpPauseAutosaveCounter>
+  <MpPauseAutosaveCounterDesc>Pauser tælleren og foryder auto-saving mens spillet er på pause.</MpPauseAutosaveCounterDesc>
+  
+  <MpLan>LAN</MpLan>
+  <MpDirect>Direkte</MpDirect>
+  <MpSteam>Steam</MpSteam>
+  <MpHostTab>Host</MpHostTab>

```

Comment:
```suggestion
  <MpHostTab>Vært</MpHostTab>
```
I believe this would be better here

Reply:
"Vært" is the person hosting something (Parties, servers and stuff like that), though im not sure if it would be incorrect either to use that instead of host but it also just sounds really stupid (Which is what makes translating all of this in to danish really fucking hard). But host is also a danish word for the same thing as host in english though it's usually only used here for companies which sell hosting space.

Reply:
Soo yes?

Reply:
Well i think it could be either as either is as correct as the other, "Vært" just sounds 100x more stupid in danish than host does.

Reply:
Really? ok well guess it stays as host if you think that danish only speakers can understand it